project/CodeCoverageSettings.scala

```diff
@@ -25,7 +25,7 @@ object CodeCoverageSettings {
 
   val settings: Seq[Setting[?]] = Seq(
     ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
-    ScoverageKeys.coverageMinimumStmtTotal := 79.2,
+    ScoverageKeys.coverageMinimumStmtTotal := 79.9,
```

Comment:
Expected min coverage can be increased to 80% as
Actual statement coverage: 80.09%
https://build.tax.service.gov.uk/job/VOA_TCTR/job/tenure-cost-and-trade-records-frontend-pr-builder/853/console